### ISSUE TITLE
[REVIEW]: catwoman: A transit modelling Python package for asymmetric light curves

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2382]
**Submitting author:** @KathrynJones1 (<a href="http://orcid.org/0000-0002-2316-6850">Kathryn Jones</a>)
**Repository:** <a href="https://github.com/KathrynJones1/catwoman" target ="_blank">https://github.com/KathrynJones1/catwoman</a>
**Version:** v1.0.12
**Editor:** @xuanxu
**Reviewer:** @jessie-dotson, @barentsen
**Archive:** <a href="https://doi.org/10.5281/zenodo.3999513" target="_blank">10.5281/zenodo.3999513</a>

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/37ce14f6fb5eae5484c5152f6f43ca99/status.svg)](https://joss.theoj.org/papers/37ce14f6fb5eae5484c5152f6f43ca99)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/37ce14f6fb5eae5484c5152f6f43ca99"><img src="https://joss.theoj.org/papers/37ce14f6fb5eae5484c5152f6f43ca99/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/37ce14f6fb5eae5484c5152f6f43ca99/status.svg)](https://joss.theoj.org/papers/37ce14f6fb5eae5484c5152f6f43ca99)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/KathrynJones1/catwoman" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@jessie-dotson & @barentsen, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @xuanxu know.

✨ **Please try and complete your review in the next six weeks** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/KathrynJones1/catwoman">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@KathrynJones1) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/KathrynJones1/catwoman">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@KathrynJones1) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @christinahedges, @barentsen it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.1086/683602 is OK
- 10.1086/345520 is OK
- 10.3847/1538-3881/aae8e5 is OK
- 10.3847/1538-4357/ab55d9 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.02382/joss.02382/10.21105.joss.02382.pdf)

**Comment by xuanxu** [comment]
@KathrynJones1, @christinahedges, @barentsen : this is the review thread for the paper. All of our communications will happen here from now on.

Both reviewers have checklists at the top of this thread with the JOSS requirements. As you go over the submission, please check any items that you feel have been satisfied. There are also links to the JOSS reviewer guidelines.

The JOSS review is different from most other journals. Our goal is to work with the authors to help them meet our criteria instead of merely passing judgment on the submission. As such, the reviewers are encouraged to submit issues and pull requests on the software repository. When doing so, please mention `openjournals/joss-reviews#2382` so that a link is created to this thread (and I can keep an eye on what is happening). Please also feel free to comment and ask questions on this thread. In my experience, it is better to post comments/questions/suggestions as you come across them instead of waiting until you've reviewed the entire package. 

We aim for reviews to be completed within about 2-4 weeks, but that's not a hard deadline. Please let me know if any of you require some more time. We can also use Whedon (our bot) to set automatic reminders if you know you'll be away for a known period of time.

Please feel free to ping me (@xuanxu) if you have any questions/concerns.

**Comment by xuanxu** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.02382/joss.02382/10.21105.joss.02382.pdf)

**Comment by xuanxu** [comment]
@jessie-dotson: you should have now an invitation [here](https://github.com/openjournals/joss-reviews/invitations) to collaborate in the repo, you have to accept it so you can edit your checklist.

**Comment by crawfordsm** [comment]
@KathrynJones1 do you have the AAS manuscript number?

**Comment by KathrynJones1** [comment]
Hi @crawfordsm , apologies, we don't have the manuscript number as we haven't submitted yet

**Comment by xuanxu** [comment]
:wave: @jessie-dotson, @barentsen how are those reviews going?

**Comment by barentsen** [comment]
@xuanxu I've been distracted by grant writing and other tasks, but expect to continue the review this weekend, and do not expect any issues carrying out the review! :+1:

**Comment by jessie-dotson** [comment]
Created https://github.com/KathrynJones1/catwoman/issues/4 to capture an easy to fix error in the tutorial.

**Comment by jessie-dotson** [comment]
Created https://github.com/KathrynJones1/catwoman/issues/5 to capture another easy to fix error in the tutorial.

**Comment by jessie-dotson** [comment]
Created https://github.com/KathrynJones1/catwoman/issues/6 to capture one more easy to fix error in the tutorial.

**Comment by jessie-dotson** [comment]
Another easy to fix tutorial error is captured in https://github.com/KathrynJones1/catwoman/issues/7

**Comment by KathrynJones1** [comment]
@jessie-dotson Thank you for finding those typos, they should all be fixed now :)

**Comment by jessie-dotson** [comment]
@KathrynJones1 awesome!  Nice job jumping right on those!  You might want to go ahead and close those issues in your catwoman github repository.  That way if anyone were following the trail from here to there, it'd be really really clear the typos were all fixed!  

Also, I opened a couple other issues in your repository.  I didn't link them here because (imho) they aren't blockers for the JOSS review, but they are suggestions motivated by the JOSS checklist I'm working through.  If they're easy to address, please consider doing so.

**Comment by jessie-dotson** [comment]
The text of the paper has a performance metric which I interpret as saying that it takes ~ 340 seconds to generate 1 million limb-darkened lightcurves each with 100 data points.  I can't reproduce that metric.   I have a different processor (2.6 GHz Intel Core i7) and am finding it takes about 4x longer.  The case I ran took 1415 seconds to create 1 million light curves with 100 data points.    Admittedly, I almost never benchmark code -- so it's *entirely* possible I didn't follow standard practices here.   The notebook showing my test is attached in this zip file.  

[timing catwoman.ipynb.zip](https://github.com/openjournals/joss-reviews/files/4984187/timing.catwoman.ipynb.zip)

**Comment by KathrynJones1** [comment]
@jessie-dotson I've had a look and I think firstly one of the reasons why your code took longer is that you set the `max_err=0.1`ppm however I calculated the timing based on the default setting of 1ppm instead (I could add this to the paper text to make it more explicit?) . It also seems you used a different timing method (time.process_time) whereas I used the timeit.default_timer() function (https://www.geeksforgeeks.org/timeit-python-examples/) which I tested actually gives a different (and lower) runtime for 1 mil lightcurves (i also found this which attempts to explain why there's a difference between the two: https://www.webucator.com/blog/2015/08/python-clocks-explained/). It seems that from what I've read the timeit.default_timer() may be better to use for timing sections of code although I admit I don't know much about this so happy to discuss in an new issue?

**Comment by barentsen** [comment]
In relation to the automated tests: I do believe it is a blocking issue that the package ships with a `catwoman.tests.test()` function which fails when I run it. Although automated tests are only strongly encouraged rather than required per JOSS guidelines, I do believe that if a package decides to ship with a `tests.test()` function then that function should succeed and be documented.

More details here: https://github.com/KathrynJones1/catwoman/issues/11#issuecomment-670287671

**Comment by KathrynJones1** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02382/joss.02382/10.21105.joss.02382.pdf)

**Comment by jessie-dotson** [comment]
Please add minimum version#s of dependencies.  See discussion in https://github.com/KathrynJones1/catwoman/issues/10

**Comment by jessie-dotson** [comment]
@KathrynJones1 -- thanks for the hint to use timeit!  I re-ran the performance test using timeit.  I still can't reproduce the quoted 300seconds, but get 400seconds instead.  I naively would've thought that my laptop (2.6 GHz Intel Core i7) would've been faster than the one quoted in the paper, but perhaps that's a naive expectation.  I also tried on an older processor (1.2 GHz Intel Core m5) and obtained ~600 seconds.  Since these are all within a factor of 2 and I don't have access to the specified processor, I'm calling it close enough...

**Comment by barentsen** [comment]
I completed my review and am happy for `catwoman` to be accepted by JOSS. :+1:

For future versions, I would encourage the authors to consider the following improvements:
* Add one or more automated unit tests.
* Expand the documentation to include more details on the following topics:
  * How to build the package from source.
  * How to contribute to the project.
  * Ideas for future improvements.

**Comment by jessie-dotson** [comment]
I also have completed my review and am happy for catwoman to be accepted by JOSS.  The future improvements identified by @barentsen would be extremely useful!  Please do keep those in mind for the future.

**Comment by xuanxu** [comment]
Great! Thanks @jessie-dotson & @barentsen!

**Comment by xuanxu** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://raw.githubusercontent.com/openjournals/joss-papers/joss.02382/joss.02382/10.21105.joss.02382.pdf)

**Comment by xuanxu** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.1086/683602 is OK
- 10.1086/345520 is OK
- 10.3847/1538-3881/aae8e5 is OK
- 10.3847/1538-4357/ab55d9 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by xuanxu** [comment]
OK @KathrynJones1, everything looks good now, here are the next steps:

- Please release a new [tagged version](https://github.com/KathrynJones1/catwoman/releases) from the current main branch so it includes all the changes made during the review process
- Then archive that latest release in Zenodo (or a similar service).
- Check the Zenodo deposit has the correct metadata: title and author names should match the paper; you may also add authors ORCID.

Once you do that please report here the version number and archive DOI

**Comment by xuanxu** [comment]
@KathrynJones1 are you having any problem or doubt about the version release/archiving process?

**Comment by KathrynJones1** [comment]
Hi @xuanxu, apologies, I have been moving countries this past week and have not got round to this yet, I will get back to you within the week with the DOI.

**Comment by KathrynJones1** [comment]
Okay so for v1.0.12, which I have labelled as the JOSS version the DOI from Zenodo is 10.5281/zenodo.3999513. Let me know if that is okay!

**Comment by xuanxu** [comment]
@whedon set v1.0.12 as version

**Comment by whedon** [comment]
OK. v1.0.12 is the version.

**Comment by xuanxu** [comment]
@whedon set 10.5281/zenodo.3999513 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.3999513" target="_blank">10.5281/zenodo.3999513</a> is the archive.

**Comment by xuanxu** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1086/683602 is OK
- 10.1086/345520 is OK
- 10.3847/1538-3881/aae8e5 is OK
- 10.3847/1538-4357/ab55d9 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/1724

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/1724, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by arfon** [comment]
@xuanxu @openjournals/joss-eics - just a reminder that we shouldn't publish this until we have the AAS (AJ) DOI.

**Comment by danielskatz** [comment]
Until then, I'm going to mark this as paused

**Comment by arfon** [comment]
/ cc @augustfly for visibility on this submission as I believe @crawfordsm is no-longer acting as software editor for AAS. (Gus - we're waiting on the AAS DOI to wrap up this submission)

**Comment by augustfly** [comment]
@arfon I'm not finding an AAS submission id for a work related to this JOSS paper.

**Comment by arfon** [comment]
OK thanks @augustfly. In [this comment](https://github.com/openjournals/joss-reviews/issues/2330#issuecomment-643191751) @KathrynJones1 suggested an AJ submission was coming.

@KathrynJones1 - can you update us on your plans here?

**Comment by KathrynJones1** [comment]
@arfon @augustfly yes we are in the final stages of preparing the other paper and plan to submit to AJ very soon. I will let you know when we do this

**Comment by xuanxu** [comment]
@KathrynJones1 please merge [this pending pull request](https://github.com/KathrynJones1/catwoman/pull/14), it adds the missing journal names to the paper.

**Comment by danielskatz** [comment]
@KathrynJones1 - please address the comment above this one.

**Comment by KathrynJones1** [comment]
Apologies, this message slipped through my notifications - I have merged the request now

**Comment by arfon** [comment]
:wave: @augustfly @KathrynJones1 - any updates on the status of the AJ paper here?

**Comment by KathrynJones1** [comment]
Apologies for the delay, we have now submitted our AJ paper - do you need a DOI or are you happy with just a submission ID?

**Comment by KathrynJones1** [comment]
I've also just edited my affiliation in the paper as it has changed recently, I hope that's okay

**Comment by augustfly** [comment]
I see the manuscript, and I am marking it as a JOSS-related submission. We would not have an article DOI until the end of the peer review process (as Arfon knows I think).

**Comment by arfon** [comment]
> I see the manuscript, and I am marking it as a JOSS-related submission. We would not have an article DOI until the end of the peer review process (as Arfon knows I think).

:+1: thanks. Yes, we just need the DOI from AAS publishing once the paper has been through review there.

**Comment by arfon** [comment]
Actually, thinking this through, I think we should probably go ahead and publish this now and then update the JOSS metadata when the AAS paper is ready. What do you all think?

**Comment by augustfly** [comment]
I'm find doing this in metadata.  This way @KathrynJones1 can update their manuscript at a later point to reference the JOSS article directly.

**Comment by KathrynJones1** [comment]
that is fine with me 👍

**Comment by arfon** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```
Reference check summary (note 'MISSING' DOIs are suggestions that need verification):

OK DOIs

- 10.1086/683602 is OK
- 10.1086/345520 is OK
- 10.3847/1538-3881/aae8e5 is OK
- 10.3847/1538-4357/ab55d9 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
:wave: @openjournals/joss-eics, this paper is ready to be accepted and published.

 Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/1923

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/1923, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by arfon** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1328743310354231300) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/1924
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.02382](https://doi.org/10.21105/joss.02382)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? Notify your editorial technical team...

**Comment by arfon** [comment]
@jessie-dotson, @barentsen - many thanks for your reviews here and to @xuanxu for editing this submission ✨. JOSS simply wouldn't work without the volunteer efforts of people like you.

@KathrynJones1 - your paper is now accepted into JOSS :zap::rocket::boom: 

Also, please let me know when the AAS paper has been accepted and you have a DOI, at that point we will associate this paper with the AAS one.

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.02382/status.svg)](https://doi.org/10.21105/joss.02382)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.02382">
  <img src="https://joss.theoj.org/papers/10.21105/joss.02382/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.02382/status.svg
   :target: https://doi.org/10.21105/joss.02382
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.02382/status.svg)](https://doi.org/10.21105/joss.02382)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss

**Comment by KathrynJones1** [comment]
@arfon @jessie-dotson @barentsen @xuanxu  🎉🎉 Fantastic! Thank you very much for all of your constructive reviews and help through the process! 
@arfon yes I will let you know 👍

**Comment by augustfly** [comment]
@KathrynJones1 Did you end up submitting an AAS Journal article that couples to this JOSS publication? Is it https://arxiv.org/abs/2106.15687?

**Comment by KathrynJones1** [comment]
Hi @augustfly , yes we did, apologies for not notifying you sooner. Yes that's the correct link :)

**Comment by augustfly** [comment]
Great!  I'll catch up with the production office to get the DOI for the AJ paper. Our production office is running very behind, but I'm guessing based on the acceptance date that you and Nestor should have the proofs by now (or soon).

**Comment by augustfly** [comment]
The AJ AAS Journal DOI will be 10.3847/1538-3881/ac134d

**Comment by KathrynJones1** [comment]
Terribly sorry, I completely lost track of this - it is done now 👍

**Comment by arfon** [comment]
@editorialbot re-accept

**Comment by arfon** [comment]
@editorialbot reaccept
